Source/EasyNetQ/Extentions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace EasyNetQ
+{
+    public static class Extentions
+    {
+        public static void SafeDispose(this IDisposable disposable)
```

Comment:
Opened on [RabbitMQ.Client](https://github.com/rabbitmq/rabbitmq-dotnet-client) issue [#119](https://github.com/rabbitmq/rabbitmq-dotnet-client/issues/119) and PR [#120](https://github.com/rabbitmq/rabbitmq-dotnet-client/pull/120)